docs/articles_en/get_started/installing-openvino-overview/installing-openvino-shared/installing-openvino-pip.md

```diff
@@ -113,7 +113,7 @@ Run the command below:
 
 .. code-block:: sh
 
-   python -c "from openvino.runtime import Core; print(Core().available_devices)"
+   python -c "from openvino import Core; print(Core().available_devices)"
```

Comment:
May it is out of scope. Will we fix https://docs.openvino.ai/2023.2/api/ie_python_api/api.html because there is still `runtime`?

Reply:
`runtime` is not removed, but most commonly used API is imported to root `openvino`